ecosystem/typescript/sdk/src/providers/Utils.ts

```diff
@@ -0,0 +1,7 @@
+import { VERSION } from "../version";
```

Comment:
can we lowercase the file name? `Utils.ts` => `utils.ts`

Reply:
Yep! But curious, is lowercase file name, the naming convention on Typescript?

Reply:
yep